app/controllers/games_controller.rb

```diff
@@ -9,7 +9,7 @@ def new
 
   def create
     @game = Game.create(game_params)
-    redirect_to root_path
+    redirect_to game_path(Game.last)
```

Comment:
@doyouthunderstand I missed this line last night :)

I'd write this as `game_path(@game)`

Just incase 2 games get created at once.